docs/README.md

```diff
@@ -0,0 +1,21 @@
+### How to generate doc
+
+```bash
+$cd python-client/docs
+$bash generate.sh
+```
+
+### How to check generated doc
+
+```bash
+$cd python-client/docs
+$bash generate.sh
+$cd python-client/docs/_build/html
+$python -m http.server 1234
+```
+
+Access to `http://localhost:1234` on web browser
+
+
+### How to deploy generated doc
+See https://github.com/ki4070ma/python-client-sphinx#how-to-deploy for now
```

Comment:
can we move that stuff into Appium org?

Reply:
yes, it makes sense to keep the stuff inside the org. @jlipps Could you please create the repository?

Reply:
repo now exists: https://github.com/appium/python-client-sphinx

Reply:
thanks Jonathan. 

@ki4070ma Would you mind to migrate the content there and update the links in the documentation?

Reply:
I'll create another PR to update the URL here after uploading docs to below repo.

https://github.com/appium/python-client-sphinx